content/api/index.html

```diff
@@ -291,6 +291,61 @@
       </div>
     </div>
 
+    <div class="int-anchor" id="metrics-metadata-get"></div><a href="#metrics-metadata-get"><h4>View metric metadata</h4></a>
+    <div class="row">
+      <%= left_side_div %>
+        <p>
+          The metrics metadata endpoint allows you to get metadata about a specific metric.
+        </p>
+
+        <h5>Arguments</h5>
+        <%= no_args %>
+      </div>
+      <%= right_side_div %>
+        <h5>Signature</h5>
+        <code>GET https://app.datadoghq.com/api/v1/metrics/:metric_name</code>
+        <h5>Example Request</h5>
+        <% %w{py sh rb}.each do |f| %>
+          <%= snippet_code_block "api-metric-metadata-get.#{f}" %>
+        <% end %>
+        <h5>Example Response</h5>
+        <% %w{py sh rb}.each do |f| %>
+          <%= snippet_result_code_block "api-metric-metadata-get.#{f}" %>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="int-anchor" id="metrics-metadata-update"></div><a href="#metrics-metadata-update"><h4>Edit metric metadata</h4></a>
+    <div class="row">
+      <%= left_side_div %>
+        <p>
+          The metrics metadata endpoint allows you to edit fields of a metric's metadata.
+        </p>
+
+        <h5>Arguments</h5>
+        <ul class="arguments">
+          <%= argument("type", "metric type such as 'gauge' or 'rate'", :lang => 'console') %>
+          <%= argument("description", "string description of the metric", :lang => 'console') %>
+          <%= argument("short_name", "short name string of the metric", :lang => 'python') %>
+          <%= argument("unit", "primary unit of the metric such as 'byte' or 'operation'", :lang => 'python') %>
```

Comment:
The lang specs for console and python don't seem correct here.

I think what you need is something like:

```
<%= argument("metric_name", "the metric name", :lang => 'python') %>
<%= argument("metric_name", "the metric name", :lang => 'ruby') %>
<%= argument("params", "the metric metadata, that includes the following:", :lang => 'python') %>
<%= argument("params", "the metric metadata, that includes the following:", :lang => 'ruby') %>
<%= argument("type", "the metric type") %>
<%= argument("description", "the metric description") %>
<%= argument("short_name", "the metric short name") %>
<%= argument("unit", "the metric unit") %>
<%= argument("per_unit", "the metric per_unit") %>
<%= argument("statsd_interval", "the metric statsd_interval") %>
```

So the metric_name and params show up when viewing python and ruby. All the params show for all langs.